apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -13,6 +13,40 @@ export function getManifest(appType, locale = 'en_us') {
   );
 }
 
+/* Returns the english default sprite list of SpriteLab
+ */
+export function getDefaultList() {
+  return fetch(`/api/v1/animation-library/default_spritelab`).then(response =>
+    response.json()
+  );
+}
+
+/* Updates the english default sprite list of SpriteLab
+ * @param listData {String} JSON object of  to upload
```

Comment:
comment nit: "JSON object of  to upload"

should we have the consumer pass a JSON object to this method, then have this method stringify that JSON? i think that step will always be necessary, so it seems easier to have this method perform it than having all consumers call `JSON.stringify`